Building-with-IntelliJ-IDEA.md

```diff
@@ -154,3 +172,7 @@ You have to add the following extra VM arguments in order to use these java vers
 ```
 --add-opens=java.desktop/com.apple.eawt=ALL-UNNAMED --add-opens=java.desktop/sun.awt=ALL-UNNAMED
 ```
+
+## Other suggestions
+
+Make sure that you don't have any changes in the code and make sure to pull the lastest changes from master. Try restarting the editor. At this point if it still doesn't work, ask for help in the [Runelite Discord](https://discord.gg/runelite).
```

Comment:
Can be removed. Adding bad code = build fails is too obvious. Pulling is explained in the section above, restarting won't do a whole lot.